boards/esp32s2-wemos-mini/include/periph_conf.h

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (C) 2022 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_esp32s2-wemos-mini
+ * @brief       Peripheral configurations for Wemos S2 mini boards
+ * @{
+ *
+ * The peripheral configurations in this file can be used for Espressif
+ * Wemos S2 minix boards that use one of the following modules:
+ *
+ * - ESP32-S2-MINI-1x module (Wemos S2 miniM-1x board)
+ * - ESP32-S2-SOLO module (Wemos S2 miniC-1x board)
+ * - ESP32-S2-WROOM module (ESP32-S2-Saola-1 board)
+ * - ESP32-S2-WROVER module (ESP32-S2-Saola-1R board)
+ *
+ * Any modifications required for specific applications
+ * can be overridden by \ref esp32_application_specific_configurations
+ * "application-specific board configuration".
+ *
+ * For detailed information about the peripheral configuration for ESP32-S2
+ * boards, see section \ref esp32_peripherals "Common Peripherals".
+ *
+ * @note
+ * Most definitions can be overridden by an \ref esp32_application_specific_configurations
+ * "application-specific board configuration" if necessary.
+ *
+ * @file
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    ADC and DAC channel configuration
+ * @{
+ */
+/**
+ * @brief   Declaration of GPIOs that can be used as ADC channels
+ *
+ * For generic boards, almost all ADC pins of ADC1 that are broken out are
+ * declared as ADC channels.
+ *
+ * @note As long as the GPIOs listed in ADC_GPIOS are not initialized as ADC
+ * channels with the `adc_init` function, they can be used for other
+ * purposes.
+ */
+#ifndef ADC_GPIOS
+#define ADC_GPIOS   { GPIO3,  GPIO5,  GPIO7,  GPIO9,  GPIO11, GPIO12, \
+                      GPIO16, GPIO18, GPIO33, GPIO35, GPIO37, GPIO39 }
+#endif
+
+/**
+ * @brief   Declaration of GPIOs that can be used as DAC channels
+ *
+ * For generic boards the 2 DAC lines GPIO17 and GPIO18 are declared as
+ * DAC channels.
+ *
+ * @note As long as the GPIOs listed in DAC_GPIOS are not initialized as DAC
+ * channels with the `dac_init` function, they can be used for other
+ * purposes.
+ */
+#ifndef DAC_GPIOS
+#define DAC_GPIOS   { GPIO17, GPIO18 }
+#endif
+/** @} */
+
+/**
+ * @name   I2C configuration
+ *
+ * For generic boards, only one I2C interface I2C_DEV(0) is defined.
+ *
+ * The GPIOs listed in the configuration are only initialized as I2C signals
+ * when module `periph_i2c` is used. Otherwise they are not allocated and
+ * can be used for other purposes.
```

Comment:
```suggestion
 * Only I2C interface I2C_DEV(0) is used.
 *
 * @note The GPIOs listed in the configuration are only initialized as I2C
 * signals when module `periph_i2c` is used. Otherwise they are not
 * allocated and can be used for other purposes.
```